seed/challenges/01-front-end-development-certification/advanced-bonfires.json

```diff
@@ -364,6 +409,15 @@
         "Las funciones que aceptan argumentos deben aceptar sólo uno, y este tiene que ser una cadena.",
         "Estos métodos deben ser el único medio para interactuar con el objeto.",
         "Recuerda utilizar <a href='//github.com/FreeCodeCamp/freecodecamp/wiki/FreeCodeCamp-Get-Help' target='_blank'>Leer-Buscar-Preguntar</a> si te sientes atascado. Intenta programar en pareja. Escribe tu propio código."
+      ],
+      "titleIt": "Crea una Persona",
+      "descriptionIt": [
+        "Completa il costruttore dell'oggetto con i metodi specificati qui sotto:",
+        "<blockquote>getFirstName()\ngetLastName()\ngetFullName()\nsetFirstName(first)\nsetLastName(last)\nsetFullName(firstAndLast)</blockquote>",
+        "Esegui i test per vedere il risultato sperato per ogni metodo.",
```

Comment:
Maybe replace "...~~sperato~~ **atteso**..." ?  Sperato (*hoped*) sounds not very 'deterministic' to me. But it's just a thought, it could work as is.